### ISSUE TITLE
Modal dialog's shade covers everything, makes page unresponsive

### ISSUE THREAD

**Issue by Chematronix** [divio/djangocms-text-ckeditor#380]
Howdy,

when I open a modal dialog, say, by double-clicking an image in the editor to look at its properties, the shade of the modal dialog covers everything, even the dialog itself, and prevents any mouse or keyboard interaction with the page.  The offending shade has the classes cke_dialog_background_cover and cms-ckeditor-dialog-background-cover.  [Screenshot here](https://lh3.googleusercontent.com/5AhC5m_vZpwwNkH3nWBLa6DIZ-zf6pSJ873Ft50ogIBQ4cj8OYvfv6baxm7k1aMnFXMkBbJbWs4FX2tf5tAV2TycLSXosmCof0ORH3i--x15q3Y7J3zquWVGObkvm6REmo3nA7Ja29a5b5Zn6Mr37Z6BRBWINTkreL5dpJfdopeQeinmolLXVgWL5s0Y9l_mDURwSEZpKqMxPmsTPS0t5NOcNi61l1Emzydpxx4e1H2HB2XShdsMbB8n4YZefm4ITIfnhIs7kFX5mw0Tg3hr57OywrATFoJVYMe_8aT26cTjmIqIfO-i3Oa3VaA0DQw5G4MBRC2_jkofsCoUjHt9KhatiyKS-eGNerZCv8UokmWc3qqEQwA-fFNL_IA3CnBfamNE9tmvkMZMxzQ4y5id4A-WJfhaSMcRZ1wQUrg_x2LMIyejGgf3auFV0K3LWz0mBLklaH2Pt_a-tT1mWlgaFp5Gz2K4Uk_mXSZYsLteYCsc03q31NRyeFiZoEuo24GTam5APL03qRD-qNkv--4hZnXek44EZ8E3sp1TPYG3PDczEOJKnkTfwEkzGV8r1bchVmXX55Wt=w1305-h943-k?v=1475014451895).

Reducing the z-index does move it behind the dialog window where it belongs, but when I close the dialog the cover is still there, still preventing any interaction.  I resorted to nuking it with display: none !important as a workaround.

Anyone else with this problem?

For bonus points, has anyone got image pasting working with Chrome in Windows?  (pasting the actual image data, not an img tag).  It works in Chromium in Linux, so I have a slight hope it can do so in Windows too...

I'm running v3.2.1 BTW. My `pip list`:
```
aldryn-bootstrap3 (1.1.1)
cmsplugin-filer (1.1.2)
dj-database-url (0.4.1)
Django (1.9.9)
django-appconf (1.0.2)
django-classy-tags (0.8.0)
django-cms (3.4.0)
django-durationfield (0.5.2)
django-filer (1.2.5)
django-formtools (1.0)
django-mptt (0.8.6)
django-polymorphic (0.8.1)
django-reversion (1.10.2)
django-sass-processor (0.4.6)
django-sekizai (0.10.0)
Django-Select2 (4.3.2)
django-treebeard (4.0.1)
django-wysiwyg (0.8.0)
djangocms-admin-style (1.2.4)
djangocms-attributes-field (0.1.1)
djangocms-bootstrap3 (0.1.1)
djangocms-cascade (0.10.0)
djangocms-column (1.6.0)
djangocms-googlemap (0.5.1)
djangocms-inherit (0.2.1)
djangocms-installer (0.8.11)
djangocms-link (1.8.2)
djangocms-style (1.7.0)
djangocms-text-ckeditor (3.2.1)
easy-thumbnails (2.3)
html5lib (0.9999999)
jsonfield (1.0.3)
libsass (0.11.1)
Pillow (3.3.1)
pip (8.1.2)
pkg-resources (0.0.0)
psycopg2 (2.6.2)
pytz (2016.6.1)
setuptools (27.3.0)
six (1.10.0)
tzlocal (1.2.2)
Unidecode (0.4.19)
webencodings (0.5)
wheel (0.29.0)
```

**Comment by vxsx** [comment]
Problem with the cover should be fixed by https://github.com/divio/djangocms-text-ckeditor/pull/374

**Comment by czpython** [comment]
@vxsx ok to close?

**Comment by vxsx** [comment]
yup

**Comment by elidickinson** [comment]
@vxsx Would you mind elaborating on how #374 works? I'm running into this same problem on my django/ckeditor integration. Does cmsdialog fix the modal background having the wrong z-index or was it something in that minified ckeditor.js file?

**Comment by vxsx** [comment]
@elidickinson sadly this is very custom to this particular installation of ckeditor. In order to make some things work on touch devices (moving dialog / resizing the area etc) we had to rewrite some of the logic of ckeditor's own dialog and resize plugins, and we didn't really do it correctly (it seems). 

What I did in the PR is to always force remove the cover before opening the dialog and after hiding it no matter what, so it always (not really) creates fresh backdrop, which leads to it's own unfortunate side effects, but we can live with that for now until proper fix is found. For example currently in Chrome the backdrop might not always be displayed, while in Firefox it always will be.

Problems with ckeditor are 80% configuration / custom hooks :(
